### PR TITLE
Chore/sc 18513/instructions to create a security injection

### PR DESCRIPTION
**Pull request by shanioffe**
Made some updates to the new security injections READMEs, just some links and additional details. No major changes.

### CODE REVIEW

File: How to create a Security Injection.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+```
+*securityChecklist.html from Integer Error CS2 Java*
+
+2. Create a new file called `checklist.js`.
+    - This program runs the checklist and is required if you want a code checklist that the user can advance through by clicking parts of the code.
+    - All of our `securityChecklist.html`'s have the same basic structure with slight variations based on the questions of the checklist. Therefore, we recommend that you copy the code from any current `checklist.js`'s and paste it into your new `checklist.js` file. From here, you can modify it to fit kinds of questions your checklist has.
+
+Example file:
+
+```java
```
Comment (bereketsega) [inline How to create a Security Injection.md]:
```suggestion
```javascript
```

File: content/Secure Coding/Input Validation/CS0 Pseudocode/checklist.js [hunk 2]
```diff
@@ -23,7 +27,18 @@ $(document).ready(function() {
...
+=======
+			// console.log('Checking all the boxes')
+			$("#"+question).prop('checked', true);	// check off question
+
+			// take focus away from current question
+			$("#"+question+"-label").removeClass("si-checklist-active");
+			$("#" + question + "-progress-label").addClass("progress-hidden");
+			$("#" + question + "-progress-label").removeClass("progress");
+
+>>>>>>> Stashed changes
```
Comment (bereketsega) [inline content/Secure Coding/Input Validation/CS0 Pseudocode/checklist.js]:
```suggestion
			$("#"+question).prop('checked', true);	// check off question

			// take focus away from current question
			$("#"+question+"-label").removeClass("si-checklist-active");
			$("#" + question + "-progress-label").addClass("progress-hidden");
			$("#" + question + "-progress-label").removeClass("progress");
```

File: content/Secure Coding/Input Validation/CS0 Pseudocode/checklist.js [hunk 3]
```diff
@@ -76,11 +91,17 @@ $(document).ready(function() {
...
 					// highlights correct answer before moving on to next question
 					setTimeout(function(){
 						previousWaitingOnCount += waitingOn.length;
 						current = advance(current);
 					}, 500);
+=======
+					previousWaitingOnCount += waitingOn.length;
+					// console.log("before advance function");
+					current = advance(current);
+>>>>>>> Stashed changes
```
Comment (bereketsega) [inline content/Secure Coding/Input Validation/CS0 Pseudocode/checklist.js]:
```suggestion
					// highlights correct answer before moving on to next question
					setTimeout(function(){
						previousWaitingOnCount += waitingOn.length;
						current = advance(current);
					}, 500);
					previousWaitingOnCount += waitingOn.length;
					// console.log("before advance function");
					current = advance(current);
```

File: resources/angular.js [hunk 4]
```diff
@@ -438,16 +438,29 @@ $scope.checkAnswers = function() {
 			for (j in unit.list) {
 				group = unit.list[j];
 				group_id = id+"-"+group.id;
+<<<<<<< Updated upstream
 
+=======
+				
+				// console.log('EACH BRO ___________________________________');
+>>>>>>> Stashed changes
```
Comment (bereketsega) [inline resources/angular.js]:
```suggestion
```

File: create.js [hunk 5]
```diff
@@ -134,41 +134,46 @@ exports.write_3 = function(name, variant, dev) {
...
+// exports.write_3("Buffer Overflow", "CS0 Java");
+=======
+// exports.write_3("Buffer Overflow", "CS0 Java"); 
 exports.write_3("Buffer Overflow", "CS0 Pseudocode");
-exports.write_3("Buffer Overflow", "CS1 C++");
-exports.write_3("Buffer Overflow", "CS1 Java");
-exports.write_3("Buffer Overflow", "CS1 Python");
-exports.write_3("Buffer Overflow", "CS2 C++");
-exports.write_3("Buffer Overflow", "CS2 Java");
+>>>>>>> Stashed changes
```
Comment (bereketsega) [inline create.js]:
```suggestion
```

File: create.js [hunk 6]
```diff
@@ -134,41 +134,46 @@ exports.write_3 = function(name, variant, dev) {
 // 'Secure Coding' Folder -------------------
 
 // exports.write_3("Best Practices for Secure Variables", "CS1 Java");
-exports.write_3("Buffer Overflow", "CS0 C++");
-exports.write_3("Buffer Overflow", "CS0 Java");
-exports.write_3("Buffer Overflow", "CS0 Python");
+// exports.write_3("Buffer Overflow", "CS0 C++");
+	dated upstream
+// exports.write_3("Buffer Overflow", "CS0 Java");
+=======
```
Comment (bereketsega) [inline create.js]:
```suggestion
```

File: content/Secure Coding/Input Validation/CS0 Pseudocode/checklist.js [hunk 7]
```diff
@@ -7,8 +7,12 @@ $(document).ready(function() {
 	var clicked = [];
 	// track which spans are needed for current question
 	var waitingOn = [];
+<<<<<<< Updated upstream
 	// helps reset prog bar for each question
 	var previousWaitingOnCount = 0;
+=======
+	let previousWaitingOnCount = 0;
+>>>>>>> Stashed changes
```
Comment (bereketsega) [inline content/Secure Coding/Input Validation/CS0 Pseudocode/checklist.js]:
```suggestion
	// helps reset prog bar for each question
	let previousWaitingOnCount = 0;
```

**Review by bereketsega — CHANGES_REQUESTED** [review]
Looks good, just small changes.

**Review by Cwagne17 — COMMENTED** [review]
After the merge comment lines are removed, looks good.

**Review by Maliyaa — APPROVED** [review]
Great! Just remove the merge lines 😄

**Review by JosueMolinaMorales — COMMENTED** [review]
Just the changes already mentioned

**Review by jouypo23 — COMMENTED** [review]
Looks good, just remove the changes requested above.

**Review by bereketsega — APPROVED** [review]

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by jouypo23 — APPROVED** [review]

**Pull request merged**